### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by shaunluttin**
This updates the README.md file with some guidelines that @thisissami and @shaunluttin created.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +64,35 @@ This guide is available in other languages too. See [Translation](#translation)
...
 
+## Rubikon Additions
+
+When importing from an index file, always use the directory name.
+
+    // good
+    import { Foo } from '../FooBarBaz';
+
+    // bad
+    import { Foo } from '..';
```
Comment (thisissami) [inline README.md]: this should be from `.`;

**Review by thisissami — CHANGES_REQUESTED** [review]

**Comment by thisissami** [comment]
feel free to merge in after you address that one thing

**Comment by thisissami** [comment]
One more thing @shaunluttin - could you add a link to your testing README from the main README? if you scroll down a tiny bit, you'll see some links to `react-testing`, `css-in-js`, etc. - it would be really useful to have the link to your new testing doc there too.

**Comment by thisissami** [comment]
also is it possible to add colors to that chart that you created for testing?

**Comment by shaunluttin** [comment]
> also is it possible to add colors to that chart that you created for testing?

It would be possible if we used an image; as far as I can tell, though, markdown in GitHub does not support colors. https://guides.github.com/features/mastering-markdown/

@thisissami 

P.S. And images have a downside of not being searchable.

**Pull request merged**